.github/workflows/python_uniffi_ci_cd.yml

```diff
@@ -32,7 +32,7 @@ env:
 jobs:
   get_version:
     runs-on: ubuntu-latest
-    outputs: 
```

Comment:
Noticed some formatting was done by your IDE. Not a big deal, but we should probably align on formatting as a team to avoid potentially going back & forth in PRs

Reply:
Yeah 100% agree.
I'll take care of this in another PR, as there will be be a bunch of whitespace related changes.